src/initial-html.js

```diff
@@ -4,206 +4,33 @@
  */
 
 export default `
-<!-- wp:image -->
```

Comment:
I guess these will be added back before merging right?

Reply:
Yes! Let's not forget to revert those changes 👍